pkg/reconciler/tenancy/workspacestatuses/README.md

```diff
@@ -0,0 +1,6 @@
+# Workspace statuses controller
+
+The controller has a single purpose - to update the workspace phase based on
+the conditions of the workspace. This way if we have an external process that
+updates the workspace, we can reflect the status of the workspace in the workspace
+object.
```

Comment:
why is this not a reconciler in tenancy/workspace ?

Reply:
Idea was that for now this is separate and easier to control via featureflag (not even installed). But I think same can be done in the workspace controller too. less writes

Reply:
will move. read the code there, Makes more sense there